resources/articles/oauth/token.md

```diff
@@ -0,0 +1,199 @@
+:title POST /oauth/token
+
+:aside
+
+## See also
+
+* [POST /oauth/ro](/oauth/ro/)
+* [POST /oauth/introspect](/oauth/introspect/)
+* [GET /oauth/userinfo](/oauth/userinfo/)
+* [GET /oauth/jwks](/oauth/jwks/)
+* [GET /oauth/authorize](/oauth/authorize/)
+* [SPiD Authentication](/authentication/)
+
+:body
+
+The `/oauth/token` endpoint is used to issue tokens to users and services.
+Users are given user tokens and services are given client/server tokens. In SPiD
+there also is 2 more token variants called admin tokens and admin client tokens,
+but these tokens are only used by internal applications like Ambassador or
+SelfService.
+
+SPiD currently supports 5 grant_types:
+
+* **client_credentials** to fetch a client/server token
+* **password** to fetch a user token, given username and password
+* **authorization_code** to exchange a one time code for a user token
+* **refresh_token** to exchange a refresh token for a new user access token
+* **urn:ietf:params:oauth:grant-type:jwt-bearer** to login with Facebook and Google+
+
+## Grant type Client credentials, for server tokens
+
+Server tokens are generally more powerful and should be kept secret. When
+fetching a token, a third parameter called `resource` is allowed, to support
```

Comment:
"third parameter": in addition to which other two? :)
Also might add reference to "Specifying a resource indicator" (as in oauth/introspect text).

Reply:
🍔